wurst/objects/units/ShopObjects.wurst

```diff
@@ -0,0 +1,36 @@
+package ShopObjects
+
+import Assets
+import UnitObjEditing
+
+import LocalObjectIDs
+
+public class ShopHeroDefinition extends HeroDefinition
+  construct(int newId, int origId)
+    super(newId, origId)
+    this
+      ..setLumberCost(0)
+      ..setFoodCost(0)
+      ..setGoldCost(0)
+      ..setStockStartDelay(0)
+      ..setStockReplenishInterval(999) // Will be programatically refilled
+      ..setStockMaximum(1)
+
+@compiletime function createDummies()
+  new ShopHeroDefinition(UNIT_RANDOM_TROLL, UnitIds.shadowhunter)
+    ..setName("Random Troll")
+    ..setTooltipBasic("Ran|c00ff00ffd|rom")
```

Comment:
Not a huge fan of this format. We don't enforce consistency right now, but in general don't highlight portions of the name. Just put the hotkey at end in its own section, e.g. `Random (D)`

Reply:
I just copy pasted the old Repick, but it didn't work because of the transparency I think, I'll fix it to be like trolls